pkg/util/steps/runner_test.go

```diff
@@ -341,3 +345,33 @@ func TestStepRunner(t *testing.T) {
 		})
 	}
 }
+
+func TestStepMetricsTopicNaming(t *testing.T) {
+	for _, tt := range []struct {
+		desc string
+		step Step
+		want string
+	}{
+		{
+			desc: "test action step naming",
```

Comment:
What does the metric topic naming look like with anonymous functions such as `func(ctx context.Context) { return successfulFunc(ctx)} `, like what we use in `pkg/cluster/install.go`?